### PR TITLE
replace EA notes with LA notes for DS content

### PR DESCRIPTION
**Pull request by imatwawana**
EA is being closed for existing iteration of dynamic sampling so EA notes need to be made LA.

### CODE REVIEW

**Review by k-fish — APPROVED** [review]

File: src/docs/product/data-management-settings/dynamic-sampling/current-limitations.mdx [hunk 1]
```diff
@@ -6,7 +6,11 @@ redirect_from:
 description: "Learn about the current limitions of dynamic sampling."
 ---
 
-<Include name="early-adopter-note.mdx" />
+<Note>
+
+This feature is in beta and is only available if your organization is participating in its limited release. Features in beta are still in-progress and may have bugs. We recognize the irony.
+
+</Note>
```
[the remark this answers is not in the document]
Comment (imatwawana) [reply src/docs/product/data-management-settings/dynamic-sampling/current-limitations.mdx:13]: > Should we have a limited-availabilty-note include or is it rare enough that it's not worth it?

This is the first time we've needed one, but it's probably not a bad idea. I'll throw one together.

File: src/docs/product/data-management-settings/dynamic-sampling/getting-started.mdx [hunk 2]
```diff
@@ -6,7 +6,11 @@ redirect_from:
 description: "Learn how to get started with dynamic sampling."
 ---
 
-<Include name="early-adopter-note.mdx" />
+<Note>
+
+This feature is in beta and is only available if your organization is participating in its limited release. Features in beta are still in-progress and may have bugs. We recognize the irony.
+
+</Note>
```
Comment (imatwawana) [inline src/docs/product/data-management-settings/dynamic-sampling/getting-started.mdx:13]:
```suggestion
<Include name="limited-avail-note.mdx" />
```

File: src/docs/product/data-management-settings/dynamic-sampling/index.mdx [hunk 3]
```diff
@@ -6,7 +6,11 @@ redirect_from:
 description: "Learn about how you can control which transactions Sentry retains using dynamic sampling."
 ---
 
-<Include name="early-adopter-note.mdx" />
+<Note>
+
+This feature is in beta and is only available if your organization is participating in its limited release. Features in beta are still in-progress and may have bugs. We recognize the irony.
+
+</Note>
```
Comment (imatwawana) [inline src/docs/product/data-management-settings/dynamic-sampling/index.mdx:13]:
```suggestion
<Include name="limited-avail-note.mdx" />
```

File: src/docs/product/data-management-settings/dynamic-sampling/sampling-configurations.mdx [hunk 4]
```diff
@@ -6,7 +6,11 @@ redirect_from:
 description: "Learn about effective dynamic sampling configurations."
 ---
 
-<Include name="early-adopter-note.mdx" />
+<Note>
+
+This feature is in beta and is only available if your organization is participating in its limited release. Features in beta are still in-progress and may have bugs. We recognize the irony.
+
+</Note>
```
Comment (imatwawana) [inline src/docs/product/data-management-settings/dynamic-sampling/sampling-configurations.mdx:13]:
```suggestion
<Include name="limited-avail-note.mdx" />
```

**Pull request merged**